trinity/main.py

```diff
@@ -179,6 +193,16 @@ class DBManager(BaseManager):
 
 @with_queued_logging
 def run_lightnode_process(chain_config: ChainConfig) -> None:
+    logger = logging.getLogger('trinity')
+    logger.info(TRINITY_HEADER)
+    logger.info(construct_trinity_client_identifier())
+    logger.info(
+        "enode://%s@%s:%s",
+        chain_config.nodekey.to_hex()[2:],
+        "[:]",
```

Comment:
It would be nice to print the actual IP this is bound to, if there is a quick way to get at it.

Reply:
This is a stub until all of this is within the `Server` class at which point this will be removed/obsolete.